package.json

```diff
@@ -91,7 +92,7 @@
     "files": [
       {
         "path": "apps/crn-frontend/build/**/*.js",
-        "maxSize": "160 kB"
+        "maxSize": "300 kB"
```

Comment:
This is lame and needs looked into. I don't know why this could be as we should be including less polyfills. I need to create a ticket and include the bundle size report so we can compare before and after to see what has been moved into the main bundle.